LPD-58844

jeffwu0724 opened the pull request.

jeffwu0724 committed:
LPD-58844 as we have changed company scope mail settings to osgi config, we do not need these files and tests any more in server admin

abesousa committed:
LPD-62555 Add method in SystemObjectDefinitionManager to getOrAddEmptyBaseModel

abesousa committed:
LPD-62555 Add getOrAddEmptyBaseModel implementations

abesousa committed:
LPD-62555 Apply usage

abesousa committed:
LPD-62555 Add tests

yuriymr committed:
LPD-55828 Adding Recycle Bin section in Space Settings

yuriymr committed:
LPD-55828 Adding new fields to SpaceSettings on Space.ts

yuriymr committed:
LPD-55828 Adding new fields on yaml

yuriymr committed:
LPD-55828 Adding new fields to AssetLibraryResourceImpl and AssetLibraryDTOConverter

abesousa committed:
LPD-62555 Fix tests

yuriymr committed:
LPD-55828 Adding validations in _deleteObjectEntry to check if the object is from CMS

yuriymr committed:
LPD-55828 Modifications to display on recycle bin only objects from spaces with the property enableTrash = true

yuriymr committed:
LPD-55828 Hiding recycle bin menu when there is no space with recycle bin enabled

mariuo committed:
LPD-63933 Implement getOrAddEmptyBaseModel AddressSystemObjectDefinitionManager

mariuo committed:
LPD-63933 Add integration test

abesousa committed:
LPD-62555 Update exception type to PortalException

abesousa committed:
LPD-62555 buildService

abesousa committed:
LPD-62555 ant build-service-portal

yuriymr committed:
LPD-55828 Adjusting tests

yuriymr committed:
LPD-55828 buildRest

yuriymr committed:
LPD-55828 Adjustments on AssetLibraryResourceImpl

yuriymr committed:
LPD-55828 Adjustments in tests

noornajj committed:
LPD-60513 Use site resource apis for vocabularies and tags

yuriymr committed:
LPD-55828 Adjusting validation on DefaultObjectEntryManagerImpl

yuriymr committed:
LPD-55828 Adding language keys

yuriymr committed:
LPD-55828 Adjustments on language keys in SpaceGeneralSettings

yuriymr committed:
LPD-55828 Adding default value to trashEnabled property on AssetLibraryDTOConverter

yuriymr committed:
LPD-55828 Adjustments on ViewRecycleBinSectionDisplayContext

marianoalvarosaiz committed:
LPD-63827 Create test to validate escaping issues.

abesousa committed:
LPD-62555 Remove unnecessary parameter

yuriymr committed:
LPD-55828 Moving the hide Recycle Bin logic frrom AssetLibraryResourceImpl to GroupModelListener

abesousa committed:
LPD-62555 Apply parameter removal to usages

yuriymr committed:
LPD-55828 Adjustments in enable Recycle Bin logic on GroupModelListener

abesousa committed:
LPD-62555 Add BaseSystemObjectDefinitionManagerTestCase

guilhermedcamacho committed:
LPD-55828 SF

guilhermedcamacho committed:
LPD-55828 buildLang

guilhermedcamacho committed:
LPD-55828 Simplify

guilhermedcamacho committed:
LPD-55828 Apply the same logic used to delete entries to the trash

guilhermedcamacho committed:
LPD-55828 Pass groupId to display context class

noornajj committed:
LPD-60513 Remove unused apis

# breaking

## What modules/apps/headless/headless-admin-taxonomy/headless-admin-taxonomy-impl/rest-openapi.yaml

Remove unnecessary apis from modules/apps/headless/headless-admin-taxonomy/headless-admin-taxonomy-impl/rest-openapi.yaml

## Why

The already existing site resource apis can be used for CMS as well

----

noornajj committed:
LPD-60513 buildREST

yuriymr committed:
LPD-55828 Obtaining groupId from themeDisplay

yuriymr committed:
LPD-55828 Reuse code

fabioafreitas committed:
LPD-58309 Add repository and dlfolder entries before race condition

noornajj committed:
LPD-60513 Fix tests

noornajj committed:
LPD-60513 Remove unused implementations

pei-jung committed:
LPD-64586 Add test to reproduce the issue

fabioafreitas committed:
LPD-58309 Add test integration to validate if repository and dlfolder entries were persisted

vngoliveira committed:
LPD-61863 Add new constant for onAfterLogin trigger

vngoliveira committed:
LPD-61863 Add onAfterLogin in the ObjectActionTrigger list

vngoliveira committed:
LPD-61863 Create a filter to display onAfterLogin only for User definition

vngoliveira committed:
LPD-61863 Create a new DestinationName to route the message from onAfterLogin

vngoliveira committed:
LPD-61863 Create a new message listener to handle the onAfterLogin trigger

vngoliveira committed:
LPD-61863 Create a new method to manage the messageBus call

vngoliveira committed:
LPD-61863 Add Unit and Integration tests

guilhermedcamacho committed:
LPD-55828 SF

fabioafreitas committed:
LPD-58309 Add override implementation of getDLFolder for code reuse

fabioafreitas committed:
LPD-58309 Baseline

fabioafreitas committed:
LPD-58309 Add implementation of getDLFolder for code reuse

joshuacords committed:
LPD-64607 headless-admin-taxonomy-impl: check vocabularyId before using

https://liferay.atlassian.net/browse/LPD-64607

joshuacords committed:
LPD-64607 asset-categories-test: import should happen even if vocabulary is missing

https://liferay.atlassian.net/browse/LPD-64607

JavierMoral committed:
LPD-64815 Added integration test to reproduce the issue

guilhermedcamacho committed:
LPD-55828 Use trash helper

guilhermedcamacho committed:
LPD-55828 Not necessary

guilhermedcamacho committed:
LPD-55828 Rename and Simplify

pedro-oliveira446 committed:
LPD-58468 Prioritize value when valueKey is populated

jeffwu0724 committed:
LPD-58844 create osgi configuration for company scope mail settings, and export the configuration for others to use

pedro-oliveira446 committed:
LPD-58468 The table’s primary key column may not match the SystemObjectDefinition primary key

jeffwu0724 committed:
LPD-58844 for test, apply usage for company scoped mail settings

jeffwu0724 committed:
LPD-58844 apply usage for company scoped mail settings

jeffwu0724 committed:
LPD-58844 migrate functional test to integration test

jeffwu0724 committed:
LPD-58844 for test, fix the broken test as we have changed to use osgi config

anabxalves committed:
LRSD-11096 - Adjusting request and components to filter out empty fields and renaming CSS classes

jeffwu0724 committed:
LPD-58844 for test, set up mail.smtp.port via reflection, and no need to call clearSession() as we provide a proxy instead of creating a real session

jeffwu0724 committed:
LPD-58857 create update process for MailSettingCompanyConfigurtaion

anabxalves committed:
LRSD-11096 - Adjustments to limit the Knowledge Article Card description

jeffwu0724 committed:
LPD-58844 SF

jeffwu0724 committed:
LPD-58844 removed unneeded properties

jeffwu0724 committed:
LPD-58844 ant update-portal-osgi-configuration-properties

rtienpal committed:
LPD-63517 Change text objectField name creation

fabioafreitas committed:
LPD-58309 Fix test due to missing instance user configuration

jeffwu0724 committed:
LPD-58844 add new method to handle migration of instance setting to osgi configuration per companyId

guilhermedcamacho committed:
LPD-55828 We should list all spaces

guilhermedcamacho committed:
LPD-55828 Adjust filter logic

guilhermedcamacho committed:
LPD-55828 Rename and simplify

guilhermedcamacho committed:
LPD-55828 New finder

guilhermedcamacho committed:
LPD-55828 buildService

joseabelenda committed:
LRSD-11096 Add line-clamp for compatibility

joseabelenda committed:
LRSD-11096 Sort

guilhermedcamacho committed:
LPD-55828 Logic refactoring

LuisOrtiz89 committed:
LPD-61506 Extracting similar logic to method

vngoliveira committed:
LPD-61863 Add new language key

vngoliveira committed:
LPD-61863 buildLang

alc-camilla committed:
LRSD-8201 Site Nav Dropdown revamp

alc-camilla committed:
LRSD-8201 SF

yuriymr committed:
LPD-55828 Adjustments on hide/show Recycle Bin logic

vngoliveira committed:
LPD-61863 Add FF

noornajj committed:
LPD-60513 baseline

guilhermedcamacho committed:
LPD-55828 If the trashEnabled property is missing, it means the setting is enabled

guilhermedcamacho committed:
LPD-55828 Simplify

guilhermedcamacho committed:
LPD-55828 Do not update if hidden value is similar

joshuacords committed:
LPD-64607 headless-admin-taxonomy-impl: check for null pointer and simplify logic

https://liferay.atlassian.net/browse/LPD-64607

guilhermedcamacho committed:
LPD-55828 Sort and use random integer for trashEntriesMaxAge

brianchandotcom committed:
LPD-62555 SF

JuanPablo-Montero committed:
LPD-65019 Add EntityModel for UtilityPage filter by Date

JuanPablo-Montero committed:
LPD-65019 Use search index when retrieving LayoutUtilityPageEntry

yuriymr committed:
LPD-55828 Adding tests

yuriymr committed:
LPD-55828 Adjustments in trash logic

guilhermedcamacho committed:
LPD-55828 Rename

guilhermedcamacho committed:
LPD-55828 Fix move to trash logic

pei-jung committed:
LPD-64586 If document is edited in publication, add the private working copy file version in publication

pedroliveirav committed:
LRSD-11103 Update external links in Liferay Sites section

shuyangzhou committed:
LPD-65522 Make Layout's C_C finder arrayable on the classPK

shuyangzhou committed:
LPD-65522 Add bulk draft layout fetching service method

shuyangzhou committed:
LPD-65522 Regenerate

shuyangzhou committed:
LPD-65522 Inline

shuyangzhou committed:
LPD-65522 Use bulk fetch for draft layout fetching

shuyangzhou committed:
LPD-65522 When the layouts are the same as ThemeDisplay layouts, just get the ThemeDisplay cached NavItems

MumenTayyem committed:
LRDOCS-14839 Object actions

MumenTayyem committed:
LRDOCS-14839 SF

MumenTayyem committed:
LRDOCS-14839 SF

MumenTayyem committed:
LRDOCS-14839 SF

MumenTayyem committed:
LRDOCS-14839 SF

MumenTayyem committed:
LRDOCS-14839 SF

MumenTayyem committed:
LRDOCS-14839 SF

MumenTayyem committed:
LRDOCS-14839 SF

MumenTayyem committed:
LRDOCS-14839 SF

MumenTayyem committed:
LRDOCS-14839 SF

MumenTayyem committed:
LRDOCS-14839 SF

MumenTayyem committed:
LRDOCS-14839 SF

MumenTayyem committed:
LRDOCS-14839 SF

MumenTayyem committed:
LRDOCS-14839 SF

robertoDiaz committed:
Revert "LPD-64517 Remove unused DepotEntryGroupRelLocalService references and related code"

This reverts commit 44cc1e1d2d5e5f6caafae31489bcb480ad2a4283.

achaparro committed:
LPD-64815 Inline

gamerson committed:
LPD-63828 debounce cluster events by adding a buffer with delay

guilhermedcamacho committed:
LPD-55828 Baseline

guilhermedcamacho committed:
LPD-55828 Extract depot-scoped object definition to set up method

guilhermedcamacho committed:
LPD-55828 Use depot-scoped object definition for recycle bin tests

rtienpal committed:
LPD-65249 Add playwright test to assert commerce products have same value input when used in object relationship

guilhermedcamacho committed:
LPD-55828 Fix unit test

rtienpal committed:
LPD-58468 Fix type for input value for null condition

guilhermedcamacho committed:
LPD-55828 Use local service

brittneyq committed:
LRCI-6342 Modify page size and change field names for testray report

brittneyq committed:
LRCI-6342 Change max previous testray case results to 5

LuisOrtiz89 committed:
LPD-61506 Rename configuration option

LuisOrtiz89 committed:
LPD-61506 ant update-portal-osgi-configuration-properties

michelevigilante committed:
LPD-65404 Fix poshi tests

abesousa committed:
LPD-65419 Refactor message to avoid duplicated action IDs

ealonso committed:
LPD-65271 Wrap the master layout content in a <div> with id="master-layout-wrapper" in view mode. This way, we won’t depend on the Liferay Theme, because currently the page editor expects a <div> with id="content" in the Liferay Theme and in some cases client have custom themes without this id causing errors.

ealonso committed:
LPD-65271 Uses new div id instead of "content" div, since it depends on the theme and customers can have custom themes without this "content" div

ealonso committed:
LPD-65271 Adds tests to assert the correct behavior

abesousa committed:
LPD-65419 Fix tests

vngoliveira committed:
LPD-61863 baseline

LuisOrtiz89 committed:
LPD-61506 buildLang

LuisOrtiz89 committed:
LPD-61506 Rename for consistency and clarity

guilhermedcamacho committed:
LPD-55828 Fix after rebase/merge

brianchandotcom committed:
LPD-55828 SF

brianchandotcom committed:
LPD-55828 Refactor and expose bug

timscottbell committed:
LPD-65446 Check against the right helm chart

carolmariaabb committed:
LPD-58468 Avoid chaining

guilhermedcamacho committed:
LPD-55828 Not necessary

timscottbell committed:
LPD-65442 Suppress verbose logging output

guilhermedcamacho committed:
LPD-55828 If no group exists, the correct filter to apply is status ANY

JavierMoral committed:
LPD-64815 Filter languageIds taking into account the available locales configured at group level

michaelhashimoto committed:
LRCI-6347 Check build url against AXIS_VARIABLE urls

brianchandotcom committed:
LPD-61506 SF

brianchandotcom committed:
LRCI-6342 SF

brianchandotcom committed:
LRCI-6342 prep next

shuyangzhou committed:
LPD-65496 Respect portal.properties configured deployment folders, which are not necessarily under liferay.home.

carolmariaabb committed:
LPD-62895 Add integration test to expose issue

carolmariaabb committed:
LPD-62895 Handle case where value is an instance of FileEntry

timscottbell committed:
LPD-65507 Provide options for cost optimization

larissacribeiro committed:
LPD-63866 Fix typo

larissacribeiro committed:
LPD-63866 Create function to extract src from img tag

larissacribeiro committed:
LPD-63866 When trying to paste/drop a file into richText field, stop event

larissacribeiro committed:
LPD-63866 Adjust existing test to cover new behavior

rtienpal committed:
LPD-58468 Add test config

brandizzi committed:
LPD-65534 Generate username/password with random_string and no upper-case/digits/special chars

oliv-yu committed:
LPD-65518 playwright: Add unique label for config input

alc-camilla committed:
LRSD-10392 - icons background

alc-camilla committed:
LRSD-10392 - fixing hover

alc-camilla committed:
LRSD-10392 - SF

guilhermedcamacho committed:
LPD-55828 Make _addDepotEntry method protected for reuse

guilhermedcamacho committed:
LPD-55828 Refactoring of method testGetCMSSectionFilterString

brianchandotcom committed:
LPD-65522 SF

brianchandotcom committed:
LRCI-6347 prep next

brianchandotcom committed:
LPD-63828 Add logging back

timscottbell committed:
LPD-65507 Setup resource requests for each template

brianchandotcom committed:
LPD-63828 Only 1 lock

brianchandotcom committed:
LPD-63828 SF

brianchandotcom committed:
LPD-63828 Wordsmith

brianchandotcom committed:
LPD-63828 Not needed

brianchandotcom committed:
LPD-63828 Wordsmith

brianchandotcom committed:
LPD-63828 Auto SF

brianchandotcom committed:
LPD-63828 Wordsmith

brianchandotcom committed:
LPD-63828 SF

brianchandotcom committed:
LPD-63828 SF

brianchandotcom committed:
LPD-63828 Auto SF

brianchandotcom committed:
LPD-63828 SF

brianchandotcom committed:
LPD-63828 Is this true?

brianchandotcom committed:
LPD-63828 Rename

achaparro committed:
LPD-65019 SF

shuyangzhou committed:
LPD-65529 Export --add-opens=java.base/java.util=ALL-UNNAMED and --add-opens=java.rmi/sun.rmi.transport=ALL-UNNAMED as JDK_JAVA_OPTIONS so that sub-process from appserver can automatically get it.

joshuacords committed:
LPD-65239 portal-search: add test

https://liferay.atlassian.net/browse/LPD-65239

guilhermedcamacho committed:
LPD-55828 Make _addDepotEntry method reusable

joshuacords committed:
LPD-65239 portal-search: allow size 0 searches for counts

https://liferay.atlassian.net/browse/LPD-65239

guilhermedcamacho committed:
LPD-55828 Refactoring of method testUpdateDepotEntry

MumenTayyem committed:
LRDOCS-14839 Refactor

MumenTayyem committed:
LRDOCS-14839 SF

MumenTayyem committed:
LRDOCS-14839 SF

MumenTayyem committed:
LRDOCS-14839 configs

MumenTayyem committed:
LRDOCS-14839 SF

sandrodw3 committed:
LPD-65271 Adapt tests

jkappler committed:
LPD-64517 Depots should only be connected to groups of type site

liferay-translation committed:
LPS-77699 Update Translations

brianchandotcom committed:
LPD-63827 Rename

ling-alan-huang committed:
LPD-65248 Auto SF

brianchandotcom committed:
LPD-64586 Match method name

brianchandotcom committed:
LPD-61863 SF

brianchandotcom committed:
LPD-64607 Wordsmith

brianchandotcom committed:
LPD-55828 Rename

brianchandotcom committed:
LPD-55828 Is this really the default depot group, or is it just a default group?

brianchandotcom committed:
LPD-55828 Auto SF

brianchandotcom committed:
LPD-55828 I almost closed the PR for this because this means you didn't read my previous commits. See b1bc238b

brianchandotcom committed:
LPD-55828 Auto SF

brianchandotcom committed:
LRDOCS-14839 This is our new pattern for oaua

brianchandotcom committed:
LRDOCS-14839 This is our new pattern for oahs

brianchandotcom committed:
LRDOCS-14839 Fix inconsistency

brianchandotcom committed:
LRDOCS-14839 oahs missed in earlier search and replace

brianchandotcom committed:
LRDOCS-14839 Is this used?

brianchandotcom committed:
LRDOCS-14839 Our usual pattern

brianchandotcom committed:
LRDOCS-14839 We only have one object action and one workflow action

brianchandotcom committed:
LRDOCS-14839 Auto SF

brianchandotcom committed:
LRDOCS-14839 Use java.util.Objects

brianchandotcom committed:
LRDOCS-14839 SF

brianchandotcom committed:
LRDOCS-14839 Rename

brianchandotcom committed:
LRDOCS-14839 Fewer dependencies

brianchandotcom committed:
LRDOCS-14839 This should work

brianchandotcom committed:
Revert "LRDOCS-14839 Is this used?"

This reverts commit 33e3934c162e4708abdc138a2e767ddf0d361eb0.

timscottbell committed:
LPD-65528 Fix rollback logic

timscottbell committed:
LPD-65595 Remove deprecated state locking mechanism

liferay-continuous-integration commented:
Test suite sf has been triggered on http://test-1-41

liferay-continuous-integration commented:
<h1>Unable to generate test report.</h1><p>Jenkins encountered the following error while generating test report:</p><pre><code>Sourced file: inline evaluation of: `` 						 							import com.liferay.jenkins.results.parser.Build; 							import co . . . '' : TargetError : at Line: 72 : in file: inline evaluation of: `` 						 							import com.liferay.jenkins.results.parser.Build; 							import co . . . '' : throw e ; 

Target exception: java.lang.RuntimeException: Using Git archive, unable to get Git working directory
</code></pre><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/"></a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-41.liferay.com/job/test-portal-source-format/921/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-41.liferay.com/userContent/jobs/test-portal-source-format/builds/921/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-41.liferay.com/job/test-portal-source-format/921">test-portal-source-format#921</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-41.liferay.com/userContent/jobs/test-portal-source-format/builds/921/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/jeffwu0724/liferay-portal/pull/862">jeffwu0724#862</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/317819183">[master] ci:test:sf - jeffwu0724 &gt; jeffwu0724 - PR#862 - 2025-09-16[12:28:02]</a><br/></div><div><strong>Testray Build ID:</strong><span></span><code>317819183</code><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-41.liferay.com/job/test-portal-source-format/921/">test-portal-source-format#921</a><br/></div></div>

joseabelenda committed:
LRSD-10392 SF

brianchandotcom committed:
LPD-60513 SF

brianchandotcom committed:
LPD-60513 Sort

brianchandotcom committed:
LPD-60513 Sort

joseabelenda committed:
LRSD-8201 SF

joseabelenda committed:
LRSD-8201 Simplify

smottal committed:
LPD-61863 Semantic Versioning

smottal committed:
LPD-55828 Semantic Versioning

jeffwu0724 committed:
LPD-58844 baseline

# breaking

## What portal-kernel/src/com/liferay/portal/kernel/util/PropsKeys.java

removed mail related portal.properties, PropsKey and PropsValue

## Why

We have migrated mail related properties to osgi config, and these portal properties are not needed any more

----

# breaking

## What portal-impl/src/com/liferay/portal/util/PropsValues.java

removed mail related portal.properties, PropsKey and PropsValue

## Why

We have migrated mail related properties to osgi config, and these portal properties are not needed any more

----
